.github/actions/label-closed-issues-with-date-v1/src/run.ts

```diff
@@ -0,0 +1,74 @@
+export default async function run(
+  core: typeof import('@actions/core'),
+  github: typeof import('@actions/github')
```

Comment:
nit - follow existing pattern: 

```suggestion
import type Core from '@actions/core'
import type Github from '@actions/github'

export default async function run(
  core: typeof Core,
  github: typeof Github
```